bears/general/FilenameBear.py

```diff
@@ -71,15 +81,33 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if new_name != filename_without_extension:
+            messages.append(
+                'Filename does not follow {} naming-convention.'.format(
+                    file_naming_convention))
+
+        if not filename_without_extension.startswith(filename_prefix):
+            new_name = filename_prefix + new_name
+            messages.append(
+                'Filename does not use the prefix {!r}.'.format(
+                filename_prefix))
+
+        if not filename_without_extension.endswith(filename_suffix):
+            new_name = new_name + filename_suffix
+            messages.append(
+                'Filename does not use the suffix {!r}.'.format(
+                filename_suffix))
```

Comment:
@Makman2 I know how to fix it.
I just wanted to show you that the error reproducible and the lines shouldn't be unindented as requested.
Tell me if I am misunderstanding the situation :)